tests/Fixer/FunctionNotation/PhpdocToReturnTypeFixerTest.php

```diff
@@ -376,6 +376,36 @@ function bar() {}
             '<?php /** @return int */ fn(): int => 1;',
             '<?php /** @return int */ fn() => 1;',
         ];
+
+        yield 'arrow function (static)' => [
+            '<?php /** @return int */ static fn(): int => 1;',
+            '<?php /** @return int */ static fn() => 1;',
+        ];
+
+        yield 'arrow function (as argument ended with ,)' => [
+            '<?php
+            Utils::stableSort(
+                $elements,
+                /**
+                 * @param int $a
+                 *
+                 * @return array
+                 */
+                static fn($a): array => [$a],
+                fn($a, $b) => 1,
+            );',
+            '<?php
+            Utils::stableSort(
+                $elements,
+                /**
+                 * @param int $a
+                 *
+                 * @return array
+                 */
+                static fn($a) => [$a],
+                fn($a, $b) => 1,
+            );',
```

Comment:
```suggestion
            '<?php
            Utils::stableSort(
                $elements,
                /**
                 * @return array
                 */
                static fn($a): array => [$a],
                fn($a, $b) => 1,
            );',
            '<?php
            Utils::stableSort(
                $elements,
                /**
                 * @return array
                 */
                static fn($a) => [$a],
                fn($a, $b) => 1,
            );',
```

`@param` is unneeded here, as this fixer handles only return type. So when it comes to reading the example, `@param` is only a noise.

Reply:
i don't think it makes any difference, OK with both